code/modules/antagonists/gang/gang.dm

```diff
@@ -151,28 +173,33 @@
 	var/list/free_clothes = list()
 	/// The specific, occupied family member antagonist datum that is used to reach the handler / check objectives, and from which the above properties (sans points) are inherited.
 	var/datum/antagonist/gang/my_gang_datum
+	/// The current theme. Used to pull important stuff such as spawning equipment and objectives.
+	var/datum/gang_theme/current_theme
+
+/// Allow gangs to have custom naming schemes for their gangsters.
+/datum/team/gang/proc/rename_gangster(datum/mind/gangster, original_name, starter_gangster)
+	gangster.current.fully_replace_character_name(gangster.current.real_name, original_name)
```

Comment:
`fully_replace_character_name` changes not only their mob's name, but all of the records associated with the mob, their pda, etc...
This includes their entry in the crew manifest
...which means anyone can see if it's a gangs round by checking for funny names on the manifest **in the lobby** before joining. Which is a bit of a metaknowledge issue

Reply:
On one hand, yeah, that's an issue. On the other hand, this prevents a lot of very annoying "Sybil Coach McDorf (as Urist McDorf)" spam for the leaders talking.